code/game/machinery/ecto_sniffer.dm

```diff
@@ -52,9 +57,7 @@
 
 
 /obj/machinery/ecto_sniffer/wrench_act(mob/living/user, obj/item/tool)
-	tool.play_tool_sound(src, 15)
-	setAnchored(!anchored)
-	balloon_alert(user, "sniffer [anchored ? "anchored" : "unanchored"]")
```

Comment:
balloon alerts were pretty cool here

Reply:
Opt'ed to use `default_unfasten_wrench` since it handles the wrench anchor/unanchor, using this means that you hit the sniffer and toggle anchor each time.